ecommerce-ios/Preview Content/Preview Assets.xcassets/icon/icon-cone.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "icon-cone.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
```

Comment:
Should we preserve vector for these?

Reply:
No, I don't think so.

Reply:
Could you explain why not?

Reply:
Because it's not a vector. If you open the PDF file with Text Editor, in case it's a vector, the text file will be definitely short with the vector instruction's code.